engines/libpmem.c

```diff
@@ -2,7 +2,7 @@
  * libpmem: IO engine that uses PMDK libpmem to read and write data
```

Comment:
I think the style is that there's an empty line above the S-o-b in the commit message.